cvmfs/cvmfs.h

```diff
@@ -46,8 +46,6 @@ std::string GetOpenCatalogs();
 unsigned GetMaxTTL();  // in minutes
 void SetMaxTTL(const unsigned value);  // in minutes
 void ResetErrorCounters();
-void GetLruStatistics(lru::Statistics *inode_stats, lru::Statistics *path_stats,
-                      lru::Statistics *md5path_stats);
 std::string PrintInodeTrackerStatistics();
 std::string PrintInodeGeneration();
 catalog::Statistics GetCatalogStatistics();
```

Comment:
This declaration should be removed (I think there is at least one more that got removed in the .cc file but not here)